govcd/certificates_embedded_test.go

```diff
@@ -1,5 +1,5 @@
-//go:build functional || openapi || certificate || alb || nsxt || ALL
-// +build functional openapi certificate alb nsxt ALL
+//go:build functional || openapi || certificate || alb || nsxt || network || ALL
+// +build functional openapi certificate alb nsxt network ALL
```

Comment:
Let's remove the double tag line. We are cleaning up in all tests with PR #558 anyway